pkg/volume/flexvolume/flexvolume.go

```diff
@@ -290,18 +290,20 @@ func (f *flexVolumeMounter) SetUpAt(dir string, fsGroup *int64) error {
 		f.options[optionKeySecret+"/"+name] = secret
 	}
 
+	glog.V(4).Infof("attempting to attach volume: %s with options %v", f.volName, f.options)
 	device, err := f.manager.attach(f)
 	if err != nil {
 		if !isCmdNotSupportedErr(err) {
-			glog.Errorf("Failed to attach volume: %s", f.volName)
+			glog.Errorf("failed to attach volume: %s", f.volName)
```

Comment:
You do not need to (and should not) lower-case the first letter of logged errors only returned errors. The reason returned errors are lower-cased is so that the message can be concatenated. Once logged there is no consumer that will attempt to concatenate it.